doc/api/cli.md

```diff
@@ -127,6 +130,16 @@ added: v12.0.0
 
 Specify the file name of the CPU profile generated by `--cpu-prof`.
 
+### `--diagnostic-dir=directory`
+
+Set the directory to which all diagnostic output files will be written to.
+Defaults to current working directory.
+
+Affects the default output directory of:
+* [--cpu-prof-dir](#--cpu-prof-dir)
```

Comment:
Considering these two dir flags are still experimental, I think we could just emit a deprecation warning for them, and make them no-ops when `--diagnostic-dir` is set (IIUC that's what this PR effectively does). Eventually we can just remove them.

Reply:
um, on the other hand, maybe we would also want to allow the user to override these directories individually through `--cpu-prof-dir` and `--heap-prof-dir` even when `--diagnostic-dir` is set?

Reply:
thats what I intented with:
```js
  if (cpu_prof && cpu_prof_dir.empty() && !diagnostic_dir.empty()) {
      cpu_prof_dir = diagnostic_dir;
    }
```